Package:     lilo
Subject:     only warning about bad partition
Author:      Joachim Wiedorn <ad_debian at joonet.de>
Forwarded:   no
Last-Update: 2013-06-07

---

diff -urN v09/src/partition.c v10/src/partition.c
--- v09/src/partition.c	2010-06-19 23:52:05.000000000 +0200
+++ v10/src/partition.c	2010-07-03 09:41:28.687586169 +0200
@@ -226,11 +226,10 @@
 	dos ? "  A DOS/Windows system may be rendered unbootable."
 		"\n  The backup copy of this boot sector should be retained."
 		: "" );
+	fprintf (errstd, "I will assume that you know what you're doing and I will proceed.\n");
 #if 0
 	if (!dos && !cfg_get_flag(cf_options,"ignore-table"))
 	    die("You may proceed by using either '-P ignore' or 'ignore-table'");
-#else
-	if (!yesno("\nProceed? ", 0)) exit(0);
 #endif
     }
     cyl = part_table[part].cyl+((part_table[part].sector >> 6) << 8);
